src/pages/Metrics/components/MetricsSimilarCard/MetricsSimilarCard.tsx

```diff
@@ -0,0 +1,62 @@
+export type SimilarPhotoType = {
+  photoId: number;
+  photoPath: string;
+  relatedColors: {
+    red: number;
+    green: number;
+    blue: number;
+  }[];
+  relatedFeatures: string[];
+};
+
+function MetricsSimilarCard({
+  photoId,
+  photoPath,
+  relatedColors,
+  relatedFeatures,
+}: SimilarPhotoType) {
+  return (
+    <li
+      className="metric-similar-card"
+    >
+      <div className="metric-similar-card__image-container">
+        <img
+          src={photoPath}
+          className="metric-similar-card__image"
+          alt="metric"
+          draggable={false}
+        />
+        <div className="metric-similar-card__colors">
+          {
+            relatedColors.map((color) => (
+              <span
+                className="metric-similar-card__color"
+                key={String(color.blue + color.red + color.green + photoId)}
+                style={{
+                  backgroundColor: `rgb(${color.red},${color.green},${color.blue})`,
+                }}
+              />
+            ))
+          }
+        </div>
+      </div>
+      <div className="metric-similar-card__features-box">
+        <p className="metric-similar-card__subtitle">Related features</p>
+        <div className="metric-similar-card__features-box-items">
```

Comment:
let's change the class to **metric-similar-card__tags** and **metric-similar-card__tag** on the children element